core/src/oauth/encryption.rs

```diff
@@ -0,0 +1,58 @@
+use anyhow::Result;
+use base64::{engine::general_purpose, Engine as _};
+use ring::{
+    aead,
+    rand::{self, SecureRandom},
+};
+use std::env;
+
+use lazy_static::lazy_static;
+
+lazy_static! {
+    static ref ENCRYPTION_KEY: aead::LessSafeKey = {
+        let encoded_key = env::var("OAUTH_ENCRYPTION_KEY").unwrap();
+        let key_bytes = general_purpose::STANDARD.decode(&encoded_key).unwrap();
+        let unbound_key = aead::UnboundKey::new(&aead::CHACHA20_POLY1305, &key_bytes).unwrap();
+        aead::LessSafeKey::new(unbound_key)
+    };
+}
+
+pub fn seal_str(s: &str) -> Result<Vec<u8>> {
```

Comment:
Given  you touch this code (even if no-op) can you add an explicit test to run before deploy.

Locally. Create a connection from `main` switch to your branch, retrieve the connection :pray: 

If sealing/unseal is inadvertently changed this would be pretty bad

Reply:
> can you add an explicit test to run before deploy

I'm not sure what you mean:
- Me doing a manual test ?
- Unit test somehow ?
- Another form of test that I'm not aware of ?

Reply:
@spolu ping in case you missed it.